### PR TITLE
Ms.signage points

### PR DESCRIPTION
**Pull request by mariano54**
Fixes issues with skipped signage points while farming

### CODE REVIEW

File: chia/full_node/full_node_store.py [hunk 1]
```diff
@@ -601,20 +637,23 @@ def new_peak(
...
                 new_eos = eos
                 break
 
-        # This cache is not currently being used
-        for sp in self.future_sp_cache.get(peak.reward_infusion_new_challenge, []):
+        future_sps: List[Tuple[uint8, SignagePoint]] = self.future_sp_cache.get(
+            peak.reward_infusion_new_challenge, []
+        ).copy()
+        for index, sp in future_sps:
+            log.warning("HERE")
```
Comment (hoffmang9) [inline chia/full_node/full_node_store.py:653]: Feels like debug left in

File: chia/full_node/full_node.py [hunk 2]
```diff
@@ -804,6 +810,61 @@ def has_valid_pool_sig(self, block: Union[UnfinishedBlock, FullBlock]):
...
         return True
 
+    async def signage_point_post_processing(
+        self,
+        request: full_node_protocol.RespondSignagePoint,
+        peer: ws.WSChiaConnection,
+        ip_sub_slot: Optional[EndOfSubSlotBundle],
+    ):
+        self.log.info(
+            f"⏲️  Finished signage point {request.index_from_challenge}/"
```
Comment (hoffmang9) [inline chia/full_node/full_node.py:820]: For just a moment I thought we were losing this but... shew...

**Review by hoffmang9 — APPROVED** [review]
Exciting change

File: chia/full_node/full_node_store.py (hunk 1 above)
Comment (mariano54) [reply chia/full_node/full_node_store.py:653]: yeah

**Pull request closed without merging**